src/utils.js

```diff
@@ -17,27 +17,55 @@ const exec = async (command, opts) => {
 
 const setupCml = async opts => {
   const { version, sudo = true } = opts;
+  const overrideRegex = /^git(?:\+(?:ssh|https))?:\/\//;
+  const versionUrl = overrideRegex.test(version);
```

Comment:
```suggestion
  const versionUrl = /^git(?:\+(?:ssh|https))?:\/\//.test(version);
```

Reply:
I'm fine with that change, but I would argue in favor of the verbosity. Easier to read/make future changes to it.

Reply:
Still want this change @casperdcl ?

Reply:
I don't mind, fine with me either way.